pact4s/src/test/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d [%thread] %-5level  %logger{35} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <logger name="io.github.jbwheatley.pact4s.Pact4sLogger" level="INFO" />
+    <logger name="au.com.dius.pact.consumer" level="DEBUG"/>
+    <logger name="au.com.dius.pact.provider" level="DEBUG"/>
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT" />
+    </root>
+</configuration>
```

Comment:
@ichengchang did you figure out if this is actually used? I feel like it probably isn't for publishing client tests. When I copied over the pact file produced by this test and ran it locally against the BPM provider test, I did see formatted output, but obviously the `logback.xml` checked in here was not used to produce this.

```
Verifying a pact between rawls-consumer and bpm-provider
  [Using File src/test/java/bio/terra/profile/service/pacts/rawls-consumer-bpm-provider.json]
  Given BPM is ok
  Request to BPM ok status
{"timestampSeconds":1687549719,"timestampNanos":276000000,"severity":"INFO","message":"Initializing Spring DispatcherServlet 'dispatcherServlet'","serviceContext":{"service":"profile","version":"0.1.163-SNAPSHOT"},"context":"default","thread":"http-nio-auto-1-exec-1","logger":"org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/]","logging.googleapis.com/sourceLocation":{"file":"org/apache/juli/logging/DirectJDKLog.java","line":173,"function":"org.apache.juli.logging.DirectJDKLog.log"}}
{"timestampSeconds":1687549719,"timestampNanos":277000000,"severity":"INFO","message":"Initializing Servlet 'dispatcherServlet'","serviceContext":{"service":"profile","version":"0.1.163-SNAPSHOT"},"context":"default","thread":"http-nio-auto-1-exec-1","logger":"org.springframework.web.servlet.DispatcherServlet","logging.googleapis.com/sourceLocation":{"file":"org/springframework/web/servlet/FrameworkServlet.java","line":525,"function":"org.springframework.web.servlet.FrameworkServlet.initServletBean"}}
{"timestampSeconds":1687549719,"timestampNanos":280000000,"severity":"INFO","message":"Completed initialization in 2 ms","serviceContext":{"service":"profile","version":"0.1.163-SNAPSHOT"},"context":"default","thread":"http-nio-auto-1-exec-1","logger":"org.springframework.web.servlet.DispatcherServlet","logging.googleapis.com/sourceLocation":{"file":"org/springframework/web/servlet/FrameworkServlet.java","line":547,"function":"org.springframework.web.servlet.FrameworkServlet.initServletBean"}}
{"timestampSeconds":1687549719,"timestampNanos":421000000,"severity":"INFO","message":"GET /status 200","serviceContext":{"service":"profile","version":"0.1.163-SNAPSHOT"},"context":"default","thread":"http-nio-auto-1-exec-1","logger":"bio.terra.common.logging.RequestLoggingFilter","logging.googleapis.com/sourceLocation":{"file":"bio/terra/common/logging/RequestLoggingFilter.java","line":98,"function":"bio.terra.common.logging.RequestLoggingFilter.doFilter"},"requestId":"axy5D52e","requestHeaders":{"host":"localhost:58195","connection":"Keep-Alive","accept-encoding":"gzip,deflate","accept":"application/json","user-agent":"Apache-HttpClient/4.5.14 (Java/17.0.2)"},"httpRequest":{"latency":"0.067s","protocol":"HTTP/1.1","requestMethod":"GET","requestSize":0,"requestUrl":"/status","responseSize":64,"status":200,"userAgent":"Apache-HttpClient/4.5.14 (Java/17.0.2)"}}
    returns a response which
      has status code 200 (OK)
      has a matching body (OK)
{"timestampSeconds":1687549719,"timestampNanos":520000000,"severity":"WARN","message":"Set the provider version using the 'pact.provider.version' property. Defaulting to '0.0.0'","serviceContext":{"service":"profile","version":"0.1.163-SNAPSHOT"},"context":"default","thread":"Test worker","logger":"au.com.dius.pact.provider.DefaultTestResultAccumulator","logging.googleapis.com/sourceLocation":{"file":"au/com/dius/pact/provider/TestResultAccumulator.kt","line":101,"function":"au.com.dius.pact.provider.DefaultTestResultAccumulator.lookupProviderVersion"}}
{"timestampSeconds":1687549719,"timestampNanos":522000000,"severity":"INFO","message":"Skipping publishing verification results for source FileSource(file=src/test/java/bio/terra/profile/service/pacts/rawls-consumer-bpm-provider.json, pact=null)","serviceContext":{"service":"profile","version":"0.1.163-SNAPSHOT"},"context":"default","thread":"Test worker","logger":"au.com.dius.pact.provider.DefaultVerificationReporter","logging.googleapis.com/sourceLocation":{"file":"au/com/dius/pact/provider/VerificationReporter.kt","line":49,"function":"au.com.dius.pact.provider.DefaultVerificationReporter.reportResults"}}

Reply:
Or maybe the logback mechanism isn't set up for when this test executes? I tried adding a dependency on `logback`, but didn't see any difference when I ran `project pact4s` locally.

Reply:
Try rename the file `logback.xml` in `pact4s/src/test/resources` directory to `logback-test.xml`.  ScalaTest uses `logback-test.xml` by default for its test runs.